Configurar dtamg-py para utilizar .env

A variável de ambiente `PYTHONIOENCODING=UTF-8` criada somente no `.env` gerou o erro 

```
dtamg-py etl-make validate -r fl_convenio_entrada_aditivo >logs/validate/fl_convenio_entrada_aditivo.json
Traceback (most recent call last):
  File "C:\Users\m752587\projects\age7\venv\Scripts\dtamg-py-script.py", line 33, in <module>
    sys.exit(load_entry_point('dtamg-py', 'console_scripts', 'dtamg-py')())
  File "c:\users\m752587\projects\age7\venv\lib\site-packages\click\core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "c:\users\m752587\projects\age7\venv\lib\site-packages\click\core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "c:\users\m752587\projects\age7\venv\lib\site-packages\click\core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\users\m752587\projects\age7\venv\lib\site-packages\click\core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\users\m752587\projects\age7\venv\lib\site-packages\click\core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\users\m752587\projects\age7\venv\lib\site-packages\click\core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "c:\users\m752587\projects\dtamg-py\dtamg_py\validate.py", line 12, in validate_cli
    validate(sys.argv[-1])
  File "c:\users\m752587\projects\dtamg-py\dtamg_py\utils.py", line 155, in validate
    json.dump(report, sys.stdout, indent=2, ensure_ascii=False)
  File "C:\Users\m752587\AppData\Local\Programs\Python\Python39\lib\json\__init__.py", line 180, in dump
    fp.write(chunk)
  File "C:\Users\m752587\AppData\Local\Programs\Python\Python39\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\x81' in position 163: character maps to <undefined>
make: *** [Makefile:56: logs/validate/fl_convenio_entrada_aditivo.json] Error 1
```

Em resumo, o json estava sendo criado com codificação windows-1252. Quando o log de validação teve acento, isso gerou o erro acima.

Depois que `PYTHONIOENCODING` foi criada diretamente como variável de ambiente no sistema o erro parou.

--- a/dtamg_py/__init__.py
+++ b/dtamg_py/__init__.py
@@ -1,0 +1,3 @@
+from pathlib import Path
+from dotenv import load_dotenv
+load_dotenv(dotenv_path=Path('.', '.env'))

--- a/dtamg_py/utils.py
+++ b/dtamg_py/utils.py
@@ -11,11 +11,8 @@ from frictionless import validate_resource
 from frictionless import validate_schema
 from dpckan import update_resource
 import pymysql
-from pathlib import Path
-from dotenv import load_dotenv
 import json
 import click
-load_dotenv(dotenv_path=Path('.', '.env'))
 
 def extract_resources(resources):
   connection = pymysql.connect(host=os.environ.get('DB_HOST'),

